src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,111 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String DATA_SPLITTER = " ";
+    private static final String COMMA_SPLITTER = ",";
+    private static final int NAME_INDEX = 0;
+    private static final int COUNT_INDEX = 1;
+    private static final String SEPARATOR = System.lineSeparator();
+    private static final int SUPLLY_INDEX = 0;
+    private static final int BUY_INDEX = 1;
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String strInFile = readFile(fromFileName);
```

Comment:
do not shorten names
```suggestion
        String fileContent = readFile(fromFileName);
```

Reply:
not fixed

Reply:
Please fix this